com.vagueentertainment.Framed.yml

```diff
@@ -0,0 +1,78 @@
+app-id: com.vagueentertainment.Framed
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: framed
+modules: 
+  
+  - name: framed
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 Framed.x86_64 /app/bin/Framed.x86_64
+      - install -D Framed.pck /app/bin/Framed.pck
+      - cp -r HoneyComb-redistributable /app/bin/HoneyComb-redistributable
+      - install -D -m755 framed.sh /app/bin/framed
+      - desktop-file-edit --remove-key=PrefersNonDefaultGPU $FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=$FLATPAK_ID $FLATPAK_ID.desktop
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - >
+        for size in {32,64,128,256}; do
+          rsvg-convert icon.svg -w "$size" -h "$size" -a -f png -o "$size.png";
+          install -Dm644 "$size.png" "/app/share/icons/hicolor/${size}x${size}/apps/$FLATPAK_ID.png";
+        done
+      
```

Comment:
installing the scalable icon should be enough

Reply:
Nice, I thought this seemed like overkill, but I saw it in another yml file and thought it was needed.